doc/ref_cert/RC2/chapters/chapter02.rst

```diff
@@ -82,7 +82,7 @@ single tests:
    in sub-sections below
 -  skip excludes the SIG labels listed as optional in `Reference
    Architecture-2 (RA-2) Chapter
-   6 <../../../ref_arch/kubernetes/chapters/chapter06.md>`__
+   6 <https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter06.md>`__
```

Comment:
Since this chapter is referenced multiple times and also in case the chapters are reorganised and the numbering changes, can we define it externally as ".. _RA2-Ch-SIG-Reqts https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter06.md" and where referenced just use "RA2-Ch-SIG-Reqts_"?

Reply:
@pgoyal01 we won't link to a markdown file at the end. It will be done via ref as soon as all docs are in rst.